### ISSUE TITLE
Missing `engines` Field Update for Node.js Version Support in v2.0.0

### ISSUE THREAD

**Issue by Phillip9587** [expressjs/body-parser#540]
Hi `body-parser` maintainers,

I noticed in the changelog for the latest release (v2.0.0) that there is a breaking change which states:

> Node.js 18 is the minimum supported version.

However, it seems that the `package.json` `engines` field was not updated to reflect this change.

Would it be possible to update the `engines` field in `package.json` to explicitly specify the range of supported Node.js versions? This would greatly help in ensuring compatibility when upgrading to `body-parser` v2.0.0.

Thank you for all your hard work on this project!

**Comment by UlisesGascon** [comment]
You are totally right. Can you create a PR for this? I will include it in the next release
